src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform, 300ms;
+
+  &__image {
+    display: block;
+    margin: auto;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    color: $default-title-color;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+
+    &-count {
+      @include onBlockDisplay;
+      font-weight: $default-font-weight;
+      font-size: 10px;
+      line-height: 14px;
+      text-align: right;
+    }
+  }
+
+  &__price {
+    @include onFlex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-label {
+      @include onBlockDisplay;
+      font-weight: $default-font-weight;
```

Comment:
If the font-weight is default can you apply it to the whole card and then just inherit it?